internal/jimm/serviceaccount/serviceaccount_test.go

```diff
@@ -23,19 +27,35 @@ type serviceAccountManagerSuite struct {
 	ofgaClient *openfga.OFGAClient
 }
 
+type credentialCopier struct{}
+
+func (cc *credentialCopier) CopyCredential(ctx context.Context, originalUser *openfga.User, newUser *openfga.User, cred names.CloudCredentialTag) (names.CloudCredentialTag, []jujuparams.UpdateCredentialModelResult, error) {
```

Comment:
The service account manager used a real "JujuManager" for it's tests but let's rather stick with a mock as I've done here.